### PR TITLE
Support arm64 platform release jobs

### PR DESCRIPTION
**Pull request by wangke19**
Support arm64 platform release jobs

### CODE REVIEW

**Comment by rioliu-rh** [comment]
stable part is missed. you can make it as empty list
e.g. https://github.com/openshift/release-tests/blob/master/_releases/ocp-4.x-test-jobs-amd64.json
because job controller will check out the stable build as well

**Review by jhuttana — COMMENTED** [review]
LGTM apart from @rioliu-rh comment to add stable part.

**Comment by wangke19** [comment]
Updated.

File: _releases/ocp-4.16-test-jobs-arm64.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "nightly": [
+    {
+      "prowJob": "periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-aws-ipi-private-f28-disasterrecovery",
+      "optional": true
+    }
+  ],
+  "stable": [
```
Comment (rioliu-rh) [inline _releases/ocp-4.16-test-jobs-arm64.json:8]: can we format it like thanks
```
{
  "nightly": [
    {
      "prowJob": "periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-aws-ipi-private-f28-disasterrecovery",
      "optional": true
    }
  ],
  "stable": []
}
```

Comment (wangke19) [reply _releases/ocp-4.16-test-jobs-arm64.json:8]: OK

**Comment by rioliu-rh** [comment]
/retitle OCPQE-24207 add 4.16 test job definition for arm64

**Pull request merged**